guidelines/taxation.md

```diff
@@ -0,0 +1,41 @@
+# Taxation
+
+## Legal environments, tax environments, and tax codes in Mews
+
+Each property (enterprise) in Mews is created with a legal and tax environment corresponding to the region in which it is located. This cannot be changed once the property has been created in Mews' database. 
```

Comment:
Shouldn't we stick with a single word - enterprise throughout the whole documentation? Also, let's not mention database, that's a technical detail, I think "This cannot be changed once the enterprise has been created in Mews." works fine.